### DISCUSSION TITLE
Manually priced services

### DISCUSSION THREAD

**Discussion by NathanRose84** [CMSgov/price-transparency-guide#459] [Q&A] [answered]
Question – Are services performed by contracted providers that need to be manually priced (i.e., there isn’t a standard rate sheet with “Service 1 - $5”) required by CMS to be included in the In-Network Negotiated Rate file?

Example:
Payer has provider agreement that pays a percentage of the provider Implant device invoice. These claims get pended and require manual intervention to reach out to the provider(s) and get the associated invoice(s). The payer is concerned about this pricing being required in the In-Network Machine Readable File since the process can’t be automated. See example contract excerpt below:

Example provider contract:
 

![image](https://user-images.githubusercontent.com/104749434/166185304-6d16eed8-b0ee-4822-b949-cabb108e5cff.png)

**Comment by sbenphia** [comment, marked as answer]
This is probably one of those where you could use the notes section provided in the new schema.  the methodology would be 100%, and the notes and or methodology would need to be identified as percent of cost rather than percent of charges, then the notes would identify that the cost is based on invoice (I've seen "manufacturer's invoice" or "paid invoice cost" or "certified invoice cost" as well because there is a little room for gaming if it is just an invoice created by an implant reseller)
